lib/packages/docutils/rstast.nim

```diff
@@ -9,7 +9,7 @@
 
 ## This module implements an AST for the `reStructuredText`:idx: parser.
 
-import strutils, json
+import strutils, json, ../../../compiler/lineinfos
```

Comment:
```suggestion
import strutils, json, "$lib/../compiler/lineinfos"
```

note: this would be wrong as it wouldn't work when bootstrapping from an external nim with --lib:lib
```
import strutils, json, "$nim/compiler/lineinfos"
```